### PR TITLE
fix: Max out modal on smaller screens -- no margin

### PR DESCRIPTION
**Pull request by ianjon3s**
# What does this PR do?

Removes margins around modal windows on smaller screens, allowing the full screen area for interactions.

### CODE REVIEW

File: editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx [hunk 1]
```diff
@@ -65,6 +65,11 @@ export const FileTaggingModal = ({
           borderTop: (theme) => `20px solid ${theme.palette.primary.main}`,
           background: "#FFF",
           margin: (theme) => theme.spacing(2),
+          height: "100%",
+          "@media screen and (max-width: 500px)": {
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx:69]:
```suggestion
          [theme.breakpoints.down("md"): {
```

I _think_ this is the right MUI syntax ([docs](https://mui.com/material-ui/customization/breakpoints/#css-media-queries)) - please test this though and don't trust me blindly! 😂

Comment (ianjon3s) [reply editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx:69]: I did try that but was given the error `Cannot find name 'theme'`. Additionally I couldn't find another instance of using theme breakpoints within `sx` styling.

Comment (DafyddLlyr) [reply editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx:69]: In order to access `theme`, you'll need to change `sx` to be a function, e.g.

```tsx
PaperProps={{
  sx: (theme) => ({
     ...
     [theme.breakpoints.down("md")]: {
       margin: 0,
       maxHeight: "none",
     },
   })
 })
 ```

Comment (DafyddLlyr) [reply editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx:69]: Docs here on this: https://mui.com/system/getting-started/the-sx-prop/#callback-values

Comment (ianjon3s) [reply editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx]: Thanks for this, updated commit incoming

File: editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx [hunk 2]
```diff
@@ -355,7 +355,7 @@ export const SelectMultiple = (props: SelectMultipleProps) => {
           },
           popper: {
             sx: {
-              boxShadow: 10,
+              boxShadow: "0 0 10px 4px rgba(0, 0, 0, 0.5)",
```
Comment (ianjon3s) [inline editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx:358]: Moving this out of `theme.ts` as it was being applied across all popper instances and breaking tooltips.

Comment (DafyddLlyr) [reply editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultiple.tsx:358]: Thanks for spotting this one

**Review by DafyddLlyr — APPROVED** [review]

**Pull request merged**